### PR TITLE
docs : Add Rapido to 'Who uses Amundsen?'

### PR DESCRIPTION
**Pull request by pPanda-beta**
### Summary of Changes
Updated Rapido in readme

### Documentation

As a side note I'm providing the existing use cases and the future roadmap.

#### Existing features: 

- We are using amundsen on top of apache atlas
- Deployment is purely on kubernetes using our custom helm chart. For some reasons we have built a fresh helm chart instead of using the official chart
- Authentication is done using via OIDC and we are using keycloak as OIDC provider. Keycloak internally uses google as identity provider.
- Authorization is done by delegating jwt to atlas instead of passing basic auth headers. Atlas is integrated with keycloak and apache ranger where we keep the centralized acls.
- Audit is also done via amundsen -> atlas -> ranger route.
- For ingestion of metadata 
  * For hive tables we have installed atlas-hive-metastore hook. Both table ddl via hive or presto goes through hive metastore, so it automatically captures it.
  * For presto views, we have extended the atlas-hive-metastore hook using [presto utility](https://bintray.com/ppanda-beta/maven/prestosql-converter) and determined additional informations which are not available at hive-metastore level. E.g. view ddl, dependency on tables, target set of columns (even for select * type of queries),  column lineage information, ... etc
  * Descriptions of table and columns we are feeding using the amundsen metadata service apis (cause it is easier than atlas v2 apis) and primarily saved on a google sheet. Once we create an automatic backup and restore system, we will remove this step, and use amundsen frontend only for metadata editing.
  * For jobs we are using Airflow-lineage plugin with atlas backend. This is helping to track which jobs are responsible  transforming data from one hdfs location to another, as well as their corresponding tables. 
  * Using programmatic descriptions we are ingesting metabase links of all tables. This is a substitution for amundsen "Preview" feature. 
  * Using programmatic descriptions we are ingesting live health charts built inside metabase. Most charts shows last 7 or 15 days health metrics. Owner of the dataset decides the visualization of the chart and the algorithm to detect anomalies. Multiple health charts for one table can be created. The integration is automated. 

#### Future roadmap :
- Fix
  * atlas search functionalities (e.g. search by column)
  * bookmark feature
  * user details feature
  * popularity score feature 
  * amundsen logout and token expire handling (e.g. with expired token search shows empty results)
  * Replace all 500 responses to frontend with correct http error codes which will be delegated from underlying microservice
- Implement
  * tags for columns, PII will be identified via tags and will be used in ranger to do column masking
  * Preview client for metabase
  * Ingest metabase dashboard metadata and display on amundsen
  * catalog powered client which provides dataframes (like spark dataframe) based on given table/view name. It will also bypass presto query engine and start from dataframes of direct hdfs paths, then apply the view sql on top of it. 
  * Update partition info on amundsen, by using _regex based partition to timestamp detector_. Current amundsen databuilder logic to identify partition watermarks are not that useful. E.g if a partition gets added today, holding 3 months old data then that should be considered as timestamp. So source of truth for partition watermark is the partition path itself, not the update timestamp. 
  * Display partition keys as metadata on amundsen frontend. This is extremely useful for the users to write optimized queries 
  * Display lineage information on amundsen sourced from atlas
  * Make hashlinks for columns on frontend. Easy to refer columns somewhere with those links. 
  * Install correct frontend error messages. e.g. ranger plugin decides access denied. 403 will pass through atlas -> amundsen metadata -> amundsen frontend (Backend) -> amundsen frontend (Browser) -> "Access denied message" 
  * User -> Team -> Dashboard -> Table relationships. (Helpful to identify consumer base for particular tables, helpful to organize access control on ranger)
  * Functional tests for entire catalog feature, involving presto, hcatalog metastore, atlas, ranger, amundsen. 

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [*] PR title addresses the issue accurately and concisely, including a title prefix.
- [*] PR includes a summary of changes.
- [*] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by pPanda-beta** [comment]
@feng-tao  @verdan 
Sorry for creating this PR so late. Please review.

**Comment by feng-tao** [comment]
thanks @pPanda-beta , the DCO fails, could you do `git commit --amend --signoff` in your local branch and repush again? also the note is very useful, are you interested in sharing your onboard experience in upcoming community meeting?

also curious: how long does it take to onboard Amundsen with how many ics in your team?

**Comment by feng-tao** [comment]
@pPanda-beta also do you think you could write up your experience as tutorial on how to boostrap Amundsen Atlas integration in https://github.com/amundsen-io/amundsen/tree/master/docs/tutorials? That will be super helpful for other Atlas users.

**Comment by pPanda-beta** [comment]
Hi @feng-tao 
I did the DCO. 
Somehow I've messed up with my github profile settings. I had to signoff with both my old and new email id, which solved the problem. 
```
Signed-off-by: Palash Das <dpalash.new@gmail.com>
Signed-off-by: Palash Das <palashkok3@gmail.com>          # << This is old mail but required to correctly sign off it
```

Referring to other points 
* It took almost 1.5 months to get the MVP done for amundsen. This includes lot of spikes, with other products, evaluating pros and cons of the solutions.  Also our MVP included atlas deployment as well, which we had to start from the docker image level. To be honest this estimation is no good for measuring actual onboarding time with amundsen. I'm sure with all the knowledges before hand and no spikes, and a ready atlas + keycloak (or any oidc) + ranger, **amundsen deployment will hardly take 3-5 days including testing**. 
* Sorry I dont get the part "how many ics in your team?" is it tech lead or product owner role? We can discuss on amundsen slack channel
* "are you interested in sharing your onboard experience in upcoming community meeting" - which points exactly you want me to share here? Like again there were lot of other associated tasks, which may not be applicable to other organisations. 
* "do you think you could write up your experience as tutorial on how to boostrap Amundsen Atlas integration in https://github.com/amundsen-io/amundsen/tree/master/docs/tutorials?" - Yes, I'd love to. Next weekend I'll add a tutorial, with some troubleshooting options.

**Comment by feng-tao** [comment]
thanks for sharing, i mostly curious how many days with how many engineers it takes to onboard atlas with Amundsen in your organization.

**Review by feng-tao — APPROVED** [review]

**Comment by pPanda-beta** [comment]
Thanks @feng-tao 

"how many days with how many engineers it takes to onboard atlas with Amundsen in your organization." 
- days : again it will differ, I believe it will take very less when we are aware of the issues, and solutions. **For specific set of versions for amundsen and atlas, it should hardly take 5-7 days to onboard it**.
- developers : We actually have a very small team for data platform. Compared to so many components that we have to enable operations of other teams. So far I was working alone on entire data catalog stream, two of my colleagues helped me to take business critical decisions on the roadmap. We also have a beta user group of size ~16, who helped us to gather feedbacks and populate metadata (descriptions, tags,...).
In short
  - Dev : 1
  - TL,PO : 1+1
  - Beta Users (+ exploratory testing): 16

Also I would like to mention that the help I got from amundsen community including you, @verdan @mgorsk1 and many others. Without your help I wont be even able to finish the first MVP of amundsen deployment.
